memory.go

```diff
@@ -20,6 +21,12 @@ func GetRamUsage() {
 //	//Implement this function to return top 5 process that are consuming the most ram
 //}
 //
-//func GetDiskUsage() {
-//	//Implement this function to display disk usage
-//}
+func GetDiskUsage() {
+	diskUsage, err := disk.Usage("/")
+	if err != nil {
+		StandardPrinter(ErrorRedColor, "Could not retrieve disk usage details.")
+	}
+	usedPercent := fmt.Sprintf("%f", diskUsage.UsedPercent)
+	ResultPrinter("Disk Usage: ", usedPercent+"%")
+	ResultPrinter("Disk Space Available: ", diskUsage.Free)
```

Comment:
try formatting the disk usage percentage to two decimal places

Reply:
done